### ISSUE TITLE
Doesn't find xcb/xkb.h in ~/.local/include

### ISSUE THREAD

**Issue by remexre** [xkbcommon/libxkbcommon#57]
On release 0.8.0, it appears that `-I` flags from `pkg-config` aren't passed in.

```
$ pkg-config --cflags xcb-xkb
-I/home/user/.local/include
$ ninja
[1/2] Compiling C object 'interactive-x11@exe/test_interactive-x11.c.o'.
FAILED: interactive-x11@exe/test_interactive-x11.c.o 
cc  -Iinteractive-x11@exe -I. -I.. -I../src -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c99 -O0 -g -fvisibility=hidden -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wpointer-arith -Wmissing-declarations -Wformat=2 -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Wbad-function-cast -Wshadow -Wlogical-op -Wdate-time -Wwrite-strings -Wno-documentation-deprecated-sync -include config.h -MMD -MQ 'interactive-x11@exe/test_interactive-x11.c.o' -MF 'interactive-x11@exe/test_interactive-x11.c.o.d' -o 'interactive-x11@exe/test_interactive-x11.c.o' -c ../test/interactive-x11.c
../test/interactive-x11.c:29:21: fatal error: xcb/xkb.h: No such file or directory
compilation terminated.
ninja: build stopped: subcommand failed.
$ ls ~/.local/include/xcb/xkb.h 
/home/user/.local/include/xcb/xkb.h
```

**EDIT**: This only seems to be true for the meson build -- if I run `autogen.sh` and then `configure && make` as usual, it installs correctly.

**Comment by bluetech** [comment]
Thanks for the report! This should be fixed 776cb52; does it work for you with this commit?

**Comment by remexre** [comment]
Looks like it does, thanks for the speedy fix
